src/main/resources/application.properties

```diff
@@ -0,0 +1,52 @@
+spring.application.name=hng-java-boilerplate
+
+# datasource configuration
+spring.datasource.url=jdbc:${connection_string}
+spring.datasource.username=${database_username}
+spring.datasource.password=${database_password}
+spring.jpa.hibernate.ddl-auto=validate
+
+# migration configuration
+spring.flyway.enabled=true
```

Comment:
remove application.properties

Reply:
@lurldgbodex , the entire file or my addition?

Reply:
@lurldgbodex , I have deleted it